src/widgets/dropdown/template.tpl

```diff
@@ -1,14 +1,44 @@
 <ul class="CDB-Dropdown-list CDB-Text CDB-Size-medium">
   <li class="CDB-Dropdown-item">
-    <button class="CDB-Dropdown-link js-toggle">Toggle widget</button>
+    <button class="CDB-Dropdown-link js-toggleCollapsed">
+      Toggle widget
+        <div>
+        <% if (collapsed) { %>
+          <input class="CDB-Toggle js-inputCollapsed" checked="checked" type="checkbox" name="collapsed">
+        <% } else { %>
+          <input class="CDB-Toggle js-inputCollapsed" type="checkbox" name="collapsed">
+        <% } %>
+          <span class="u-iBlock CDB-ToggleFace"></span>
+        </div>
+    </button>
   </li>
   <li class="CDB-Dropdown-item">
-    <button class="CDB-Dropdown-link js-pin">Pin widget</button>
+    <button class="CDB-Dropdown-link js-togglePinned">
+        Pin widget
+        <div>
+          <% if (pinned) { %>
+            <input class="CDB-Toggle js-inputPinned" checked="checked" type="checkbox" name="pinned">
+          <% } else { %>
+            <input class="CDB-Toggle js-inputPinned" type="checkbox" name="pinned">
+          <% } %>
+          <span class="u-iBlock CDB-ToggleFace"></span>
+        </div>
+    </button>
   </li>
 
-  <% if (normalizeHistogram) { %>
+  <% if (flags.normalizeHistogram) { %>
```

Comment:
I'd try to use the necessary variables from the view and not using `var.whatever`.